exporters/pelorus/deserialization/__init__.py

```diff
@@ -414,9 +500,15 @@ def _deserialize_list(
         if not errors:
             return list_
         else:
+            unstructured_src = self.unstructured_data_path[-1]
+            src_name = (
+                unstructured_src
+                if isinstance(unstructured_src, str)
+                else format_path(unstructured_src)
+            )
             raise DeserializationErrors(
                 errors,
-                src_name=self.unstructured_data_path[-1],
+                src_name=src_name,
                 target_name=self.structured_field_name_path[-1],
             )
 
```

Comment:
I would separate this file in small ones (or just another one) and just have an `__all__` here, just because this is a `__init__` file

Reply:
The only reason this is an `__init__` instead of just `deserialization.py` is to put errors somewhere else, just to keep things a little more manageable. I don't know if `__init__` needs to just be re-exports. [Even the python stdlib has meaningful code in `__init__` sometimes!](https://github.com/python/cpython/blob/702a5bc4637c82dc011e98b84f0cede98eb08dda/Lib/json/__init__.py)